h/templates/panels/navbar.html.jinja2

```diff
@@ -24,6 +25,9 @@
 {% endmacro %}
 
 <header class="nav-bar">
+  <template id="lozenge-template">
```

Comment:
This is a template for client rendered lozenges. On the client, we clone this each time a new lozenge is created.